src/components/Carousel.js

```diff
@@ -546,12 +548,10 @@ class Carousel extends Component {
     const animationSpeed = this.getProp('animationSpeed');
     const transitionEnabled = this.state.transitionEnabled;
     const draggable = this.getProp('draggable') && children && children.length > 1;
-
```

Comment:
could you revert these lines (an empty line where it was and no empty line where there was no empty line)?

Reply:
thanks for reverting next lines but could you also revert empty line here?

Reply:
yeah done. I missed in the earlier commit.